ffi/custom_passes.cpp

```diff
@@ -864,6 +887,10 @@ struct RefPrunePass : public FunctionPass {
             while (workstack.size() > 0) {
                 // Get a basic block
                 BasicBlock *cur_node = workstack.pop_back_val();
+                // If cur_node is a raising block, then skip it
+                if (isRaising(cur_node)) {
+                    continue;
+                }
```

Comment:
This needs to respect the `prune_raise_exit` flag from `findFanout()`.

Reply:
addressed this suggestion. Now check `prune_raise_exit` as well.